### PR TITLE
VGWH NetRS to NetR9, Zephyr 1 to Zephyr 2.

### PR DESCRIPTION
**Pull request by staylorofford**

### CODE REVIEW

File: install/firmware.csv [hunk 1]
```diff
@@ -877,7 +877,8 @@ Trimble Navigation Ltd.,TRIMBLE NETR9,5505R50175,4.93,2015-07-23T00:00:00Z,9999-
 Trimble Navigation Ltd.,TRIMBLE NETR9,5506R50082,4.93,2015-07-23T00:00:00Z,2017-04-07T01:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5506R50082,5.15,2017-04-07T01:00:01Z,2017-06-20T17:21:11Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5506R50082,5.22,2017-06-20T17:21:12Z,9999-01-01T00:00:00Z,enable GPS-L5
-Trimble Navigation Ltd.,TRIMBLE NETR9,5508R50018,4.93,2015-07-23T00:00:00Z,9999-01-01T00:00:00Z,
+Trimble Navigation Ltd.,TRIMBLE NETR9,5508R50018,4.93,2015-07-23T00:00:00Z,2018-05-04T01:35:01Z,
+Trimble Navigation Ltd.,TRIMBLE NETR9,5508R50018,5.22,2018-05-04T01:35:02Z,9999-01-01T00:00:00Z,enable GPS-L5
```
Comment (elidana) [inline install/firmware.csv]: the firmware on the receiver is still on 5.14. 
Can you please modify this here (and, when you have time, update the firmware on the receiver and do another pr to reflect that?). Thanks

```
Receiver Type: 	NetR9
System Name: 	VGWH
Serial Number: 	5508R50018
Ethernet IP: 	192.168.220.88
Firmware Version: 	5.14
```

**Comment by staylorofford** [comment]
Added firmware changes in a separate commit

File: install/firmware.csv [hunk 2]
```diff
@@ -877,7 +877,8 @@ Trimble Navigation Ltd.,TRIMBLE NETR9,5505R50175,4.93,2015-07-23T00:00:00Z,9999-
 Trimble Navigation Ltd.,TRIMBLE NETR9,5506R50082,4.93,2015-07-23T00:00:00Z,2017-04-07T01:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5506R50082,5.15,2017-04-07T01:00:01Z,2017-06-20T17:21:11Z,
 Trimble Navigation Ltd.,TRIMBLE NETR9,5506R50082,5.22,2017-06-20T17:21:12Z,9999-01-01T00:00:00Z,enable GPS-L5
-Trimble Navigation Ltd.,TRIMBLE NETR9,5508R50018,4.93,2015-07-23T00:00:00Z,9999-01-01T00:00:00Z,
+Trimble Navigation Ltd.,TRIMBLE NETR9,5508R50018,4.93,2015-07-23T00:00:00Z,2018-05-04T00:00:00Z,
+Trimble Navigation Ltd.,TRIMBLE NETR9,5508R50018,5.14,2018-05-04T00:00:01Z,2018-05-07T02:33:03Z,enable GPS-L5
```
Comment (elidana) [inline install/firmware.csv]: sorry to be a pain! the GPS-L5 enabling comes with fw 5.22.
No need to have two different commits, as long as the firmware timestamps/lists are correct in the firwmare.csv file
(sorry for the misunderstanding, that was my bad english)

but...can you please change the file so that the L5 enabling is coming with 5.22 (and not 5.14) for this receiver?
Thanks

**Comment by staylorofford** [comment]
All changes made now in 1 commit
